pkg/sdn/plugin/node_iptables.go

```diff
@@ -95,6 +95,7 @@ func (n *NodeIPTables) getStaticNodeIPTablesRules() []FirewallRule {
 		{"filter", "INPUT", []string{"-p", "udp", "-m", "multiport", "--dports", VXLAN_PORT, "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", TUN, "-m", "comment", "--comment", "traffic from SDN", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", "docker0", "-m", "comment", "--comment", "traffic from docker", "-j", "ACCEPT"}},
+		{"filter", "FORWARD", []string{"-s", n.clusterNetworkCIDR, "-m", "conntrack", "--ctstate", "INVALID", "-j", "DROP"}},
```

Comment:
@danwinship So, where do we go from here? It seems risky to drop the FORWARD -s clusterNetworkCIDR rule without an understanding of why it was there in the first place and a lot of testing. Adding the drop rule for the invalid packets seems very limited in scope with minimal risk of breaking something.
CC @knobunc